PhoneNumberKit/UI/TextField.swift

```diff
@@ -72,7 +72,7 @@ open class PhoneNumberTextField: UITextField, UITextFieldDelegate {
     
     private var _partialFormatter: PartialFormatter?
 
-    private var partialFormatter: PartialFormatter {
```

Comment:
For this you could actually just use 

```swift
public private(set) var partialFormatter: PartialFormatter
```

Reply:
No, I could not write in this way. I already wrote about this.

The property `self.partialFormatter` must be initialized only after `super.init` call so it cannot be declared as `public private(set) var partialFormatter` because so it should be declared before   `super.init` call (where computed property `defaultRegion` cannot be accessed).

Reply:
Example:
```swift
    public private(set) var partialFormatter: PartialFormatter
    override public init(frame: CGRect) {
        super.init(frame:frame)
        self.partialFormatter = PartialFormatter(phoneNumberKit: phoneNumberKit, defaultRegion: defaultRegion, withPrefix: withPrefix)
    }
```
Error: Property 'self.partialFormatter' not initialized at super.init call

```swift
    public private(set) var partialFormatter: PartialFormatter
    override public init(frame: CGRect) {
        self.partialFormatter = PartialFormatter(phoneNumberKit: phoneNumberKit, defaultRegion: defaultRegion, withPrefix: withPrefix)
        super.init(frame:frame)
    }
```
Error: 'self' used in property access 'defaultRegion' before 'super.init' call

Reply:
Because of this: `PartialFormatter` is declared as optional. Before `super.init` `PartialFormatter` is `nil` and after has a value.
And for backwards-compatible, I declared a getter that unwraps `PartialFormatter?`.

Reply:
All my changes are backward compatible, if you think that something is strange, try to think in this way. And of course, in the next major versions, all must be clear

Reply:
How about `public private(set) lazy` that would definitely work in your case.

Reply:
The same `property 'self.partialFormatter' not initialized at super.init call`

Reply:
sorry, all OK with this pretty solution

Reply:
No worries mate, I'll be waiting for the other reviewers first before we can merge this.  Thank you for cooperating 😄